docs/source/getting-started/tabular-data/index.rst

```diff
@@ -0,0 +1,103 @@
+:orphan:
+:nosearch:
+
+Working with Tabular Data
+=========================
+
+Motivation 
+----------
+
+In the following examples, we will use real data to demonstrate the utility of OpenDP functions.
+We will use the `Labour Force Survey microdata <https://ec.europa.eu/eurostat/web/microdata/public-microdata/labour-force-survey>`_ released by Eurostat for a few reasons: 
+
+1. **Generality:** The dataset is relatively general, with variables and contexts accessible to users across various domains.
+2. **Sample Utility:** The public microdata is a sample of the private, full microdata. Methods developed with the public microdata will also work on the private microdata, and researchers can request access to the full dataset through Eurostat. 
+3. **Realism**: Since the dataset tracks individuals over multiple years, we need to think more carefully about our unit of privacy.
```

Comment:
More explanation needed. Why are we calling this particular issue "realism" and not any other one? Other issues/properties are also "real". Also, why does having individuals in the dataset over multiple years demand more careful thinking about the unit of privacy? More explanation on that would be better.

Reply:
could you expand on this a bit? I added a bit more here: 

3. **Realism**: This is a real dataset that tracks individuals over multiple years, which will influence the unit of privacy since each individual can be represented multiple times in the dataset.